packages/rest/src/__tests__/unit/rest.application/rest.application.unit.ts

```diff
@@ -50,5 +50,13 @@ describe('RestApplication', () => {
         ERR_NO_MULTI_SERVER,
       );
     });
+
+    it('when attempting to redirect endpoint', () => {
```

Comment:
This test is nested under `describe('throws')`, which is a block of tests verifying conditions where `RestApplication` is expected to throw an error. Your test does not throw any error though, thus it should be moved outside of the "throws" block.

Also please improve the test name to say what's the expected result. For example:

```ts
it('binds a redirect route for each "server.redirect()" call', () => {
```

I also find it suspicious that you are calling `server.redirect` in a test for `RestApplication`. I think you should call `app.redirect` instead and update the test name accordingly.

Reply:
my mistake , a new file created to unit test for redirect . 
i think this will be helpful for the next pull request when we gonna test
 the new functionality